Documentation/api.md

```diff
@@ -188,6 +192,62 @@ BasicAuth allow an endpoint to authenticate over basic authentication More info:
 
 [Back to TOC](#table-of-contents)
 
+## BlackboxExporterSpec
+
+BlackboxExporterSpec contains specification parameters for the BlackboxExporter used for probing.
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| scheme | HTTP scheme to use for scraping. | string | false |
+| url | The URL of the blackbox exporter. | string | true |
+| path | HTTP path to scrape for metrics. | string | false |
+
+[Back to TOC](#table-of-contents)
+
+## BlackboxMonitor
+
+BlackboxMonitor defines monitoring for a set of static targets or ingresses.
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| metadata | Standard object’s metadata. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata | [metav1.ObjectMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#objectmeta-v1-meta) | false |
+| spec | Specification of desired Ingress selection for target discovery by Prometheus. | [BlackboxMonitorSpec](#blackboxmonitorspec) | true |
+
+[Back to TOC](#table-of-contents)
+
+## BlackboxMonitorList
+
+BlackboxMonitorList is a list of BlackboxMonitors.
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| metadata | Standard list metadata More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#metadata | [metav1.ListMeta](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#listmeta-v1-meta) | false |
+| items | List of BlackboxMonitors | []*[BlackboxMonitor](#blackboxmonitor) | true |
+
+[Back to TOC](#table-of-contents)
+
+## BlackboxMonitorSpec
+
+BlackboxMonitorSpec contains specification parameters for a BlackboxMonitor.
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| jobLabel | The label to use to retrieve the job name from. | string | false |
+| blackboxExporterSpec | Specification for the blackbox exporter to use for probing targets. | [BlackboxExporterSpec](#blackboxexporterspec) | false |
+| module | The module to use for probing. | string | false |
+| staticTargets | List of static targets for the blackbox exporter. | []string | false |
+| selector | Select Ingress objects by labels. | [metav1.LabelSelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#labelselector-v1-meta) | false |
+| namespaceSelector | Select Ingress objects by namespace. | [NamespaceSelector](#namespaceselector) | false |
+| params | Optional HTTP URL parameters | map[string][]string | false |
+| interval | Interval at which metrics should be scraped | string | false |
+| scrapeTimeout | Timeout after which the scrape is ended | string | false |
+| honorLabels | HonorLabels chooses the metric's labels on collisions with target labels. | bool | false |
```

Comment:
I don't think honoring labels or timestamps is ever necessary when blackbox probing no?

Reply:
Right. Removed.

Reply:
Not trying to be pushy at all, but the comment suggests it's removed but I'm not seeing new/changed commits. Did you forget to push? (if not that's totally fine, all I'm trying to make sure is that if there is a commit sitting locally on your computer to have it be pushed for review :) )

Reply:
I forgot to push. Did so now 😅 
Will add one more since there's one comment not addressed

Reply:
Awesome! Looking forward to it!

Reply:
Done